contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1051,6 +1051,9 @@ class tsqlBuilder : public tsqlCommonMutator
     tree::ParseTree *parser;
     MyInputStream &stream;
 
+	std::vector<int> double_quota_places;
```

Comment:
If it is read-only, can we use `const std::vector<int> *` or `const std::vector<int>` with `std::move`?

Reply:
we initialize with 
````
std::vector<int> &quota_places
````

it's the same with : 
````
const std::vector<std::string> &rules
````

Reply:
my suggestion was to add `const`